src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -304,7 +304,14 @@ private static string GetSize(long length)
         private static Option ProvidersOption() =>
             new Option(
                 alias: "--providers",
-                description: @"A list of EventPipe providers to be enabled. This is in the form 'Provider[,Provider]', where Provider is in the form: 'KnownProviderName[:Flags[:Level][:KeyValueArgs]]', and KeyValueArgs is in the form: '[key1=value1][;key2=value2]'. These providers are in addition to any providers implied by the --profile argument. If there is any discrepancy for a particular provider, the configuration here takes precedence over the implicit configuration from the profile.",
+                description: @"A comma delimitted list of EventPipe providers to be enabled. This is in the form 'Provider[,Provider]'," +
+                             @"where Provider is in the form: 'KnownProviderName[:[Flags][:[Level][:[KeyValueArgs]]]]', and KeyValueArgs is in the form: " +
+                             @"'[key1=value1][;key2=value2]'.  Values in KeyValueArgs that contain ';' or '=' characters need to be surrounded by '""', " +
+                             @"e.g., FilterAndPayloadSpecs=""MyProvider/MyEvent:-Prop1=Prop1;Prop2=Prop2.A.B;"".  Depending on your shell, you may need to " +
```

Comment:
Is there a way to break this up into paragraphs or something? I think it's very overwhelming when rendered on the console :)

Reply:
I tried to inject `\n`, but S.CommandLine seems to strip all formatting or extra whitespace in the help text.

Reply:
Have you chatted with @jonsequitur to see if we have other options?

Reply:
You can implement a custom `IHelpBuilder` which will let you completely take control of the help output. There's a newer version of `System.CommandLine` as well, which makes some of the existing help builder functionality available for reuse. 

It also provides more information in the help and synopsis about whether arguments are required, and more control over argument names.

Reply:
Chatting with Jon offline about this.  Since there are a number of changes that are waiting on updating the S.CommandLine, it might make sense to batch "better help text formatting" with those changes.  I'll figure out if there is a way to do it with the version we are currently using though.